src/components/ScrollTrack/ScrollTrack.js

```diff
@@ -56,6 +59,14 @@ class ScrollTrack extends Component {
     window.removeEventListener('resize', this.debouncdComputeSlideAttributes)
   }
 
+  get childrenWithTrackProps() {
+    const nodeWidths = this.getNodeWidths()
+
+    return React.Children.map(this.props.children, child => {
+      return React.cloneElement(child, { ...this.state, ...nodeWidths })
+    })
+  }
```

Comment:
Should we namespace these? They're somewhat generically named within the component, and may collide with the children's own props.